src/Xamarin.Android.Tools.AndroidSdk/JdkInfo.cs

```diff
@@ -258,6 +259,12 @@ static bool AnySystemJavasInstalled ()
 					const string NameValueDelim         = " = ";
 					if (string.IsNullOrEmpty (e.Data))
 						return;
+					if (e.Data.StartsWith ("Property settings:", StringComparison.Ordinal)) {
+						foundPS = true;
+					}
+					if (!foundPS) {
+						return;
```

Comment:
I think if we at least logged a message here, that would be helpful. It should show up in diagnostic MSBuild logs?

Reply:
Related: log *where*?  Unlike [`AndroidSdkInfo`](https://github.com/xamarin/xamarin-android-tools/blob/dac3a4740d6437ff9b2d7bb8b34516364c85ac06/src/Xamarin.Android.Tools.AndroidSdk/AndroidSdkInfo.cs#L13), there is no `Action<TraceLevel, string>` parameter.  We could certainly add one, but that would require changes to calling code to receive those messages, otherwise they'd just go to stdout/stderr.

Reply:
Seems like we're good then.